scripts/docker/enable-winrm.ps1

```diff
@@ -15,7 +15,11 @@ if (Test-Path A:\install-containers-feature.ps1) {
 Stop-Service winrm
 . sc.exe config winrm start= delayed-auto
 
+Set-Service winrm -startuptype "auto"
+
 netsh advfirewall firewall set rule group="Windows Remote Administration" new enable=yes
 netsh advfirewall firewall set rule name="Windows Remote Management (HTTP-In)" new enable=yes action=allow
 
-Restart-Computer
+Restart-Service winrm
```

Comment:
We really need a reboot before letting packer login and running some more provision scripts.